deepchem/feat/bert_featurizer.py

```diff
@@ -0,0 +1,65 @@
+from deepchem.feat.base_classes import Featurizer
+from deepchem.feat.molecule_featurizers import RobertaFeaturizer
+from typing import Iterable
+import numpy as np
+import logging, warnings
+logger = logging.getLogger(__name__)
+try:
+  from transformers import BertTokenizerFast, RobertaTokenizerFast
+except:
+  raise ImportError("""This class requires the transformers package,
+                    which was not found in your environment.""")
+# TODO discuss RDKitMol support (perhaps RobertaFeaturizer should not be a moleule featurizer?
+# Current approach to handling both molecules and strings is a bit messy.
+try:
+  from deepchem.utils.typing import RDKitMol
+except:
+  warnings.warn("""RDKitMol was not found in your environment, so this class
+                will only featurize strings and will not featurize RDKitMol
+                objects.""")
+                
+
+class BertFeaturizer(RobertaFeaturizer, BertTokenizerFast):
+  """Bert Featurizer.
+
+  The Bert Featurizer is a wrapper class for HuggingFace's BertTokenizerFast.
+
+  This class intends to allow users to use the BertTokenizer API while
+  remaining inside the DeepChem ecosystem.
+
+  Usage Example (based on HuggingFace and RostLab ProtBert docs)
```

Comment:
This should just be
```
Examples
--------
```
Check out the numpy doc style https://numpydoc.readthedocs.io/en/latest/format.html